ground/gcs/src/plugins/telemetryscheduler/telemetryschedulergadgetwidget.cpp

```diff
@@ -1089,7 +1089,6 @@ void QFrozenTableViewWithCopyPaste::init()
                                    "color: #FF0000;"
                                    "background-color: #8EDE21;"
                                    "selection-background-color: #999}"); //for demo purposes
-    frozenTableView->setSelectionModel(selectionModel());

```

Comment:
Was the telemetry scheduler tested after this change? Do you recall what was the warning Qt was giving you?